install-clients.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
I expect the `pip uninstall` and `R CMD REMOVE` to fail when the user does not already have the packages installed.  In that case, I am okay with the error and I'd like the script to move on to installing the packages anyway.  If both are not installed, you will see this output:
```
me@mbp: scripts ✨  ./install-clients.sh
Cannot uninstall requirement h2o, not installed
Processing /Users/me/h2oai/github/h2o-3/h2o-py/dist/h2o-3.13.0.99999-py2.py3-none-any.whl
Requirement already satisfied: future in /usr/local/lib/python2.7/site-packages (from h2o==3.13.0.99999)
Requirement already satisfied: requests in /usr/local/lib/python2.7/site-packages (from h2o==3.13.0.99999)
Requirement already satisfied: tabulate in /usr/local/lib/python2.7/site-packages (from h2o==3.13.0.99999)
Requirement already satisfied: colorama>=0.3.8 in /usr/local/lib/python2.7/site-packages (from h2o==3.13.0.99999)
Installing collected packages: h2o
Successfully installed h2o-3.13.0.99999
Removing from library ‘/Library/Frameworks/R.framework/Versions/3.3/Resources/library’
Error in find.package(pkgs, lib) : there is no package called ‘h2o’
Execution halted
* installing to library ‘/Library/Frameworks/R.framework/Versions/3.3/Resources/library’
* installing *source* package ‘h2o’ ...
** R
** demo
** inst
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
* DONE (h2o)
```